rules/0330-sysmon_rules.xml

```diff
@@ -239,6 +239,182 @@
         <group>sysmon_event_15,</group>
     </rule>
 
+        <!--
+    -  Process anomalies based on Sysmon Event 1
+    -  Author: Josh Brower, Josh@DefensiveDepth.com.
+    -  Licensed under the MIT License: http://opensource.org/licenses/MIT
+    -  Windows Process Attributes documentation here: http://defensivedepth.com/windows-processes
+    -  Updated by Wazuh, Inc. <support@wazuh.com>.
+-->
+
+    <rule id="20350" level="12">
```

Comment:
This set of rules is not included in the group `sysmon_process-anomalies` as the others.